cmd/atlas/internal/cmdapi/cmdapi.go

```diff
@@ -23,6 +23,32 @@ var (
 		Use:          "atlas",
 		Short:        "A database toolkit.",
 		SilenceUsage: true,
+		PostRunE: func(cmd *cobra.Command, _ []string) error {
```

Comment:
Considering we wanna add some information to the update, like "this fixes a critical bug", don't we wanna check before the CLI work instead if after it?

Reply:
Which tools? My thought here is to not block the operation the user wants to perform.